math/src/field/fields/u64_prime_field.rs

```diff
@@ -281,4 +292,73 @@ mod tests {
         let f2 = FE::new(f1.representative());
         assert_eq!(f1, f2);
     }
+
+    #[test]
+    fn generate_random_using_rng_generator_and_check_if_normal_distribution() {
```

Comment:
Aren't PRNGs expected to be uniform rather than normal?

Reply:
This is extracted from the linked documentation:

> By default, random numbers in the rand crate have [uniform distribution](https://en.wikipedia.org/wiki/Uniform_distribution_(continuous)). The [rand_distr](https://docs.rs/rand_distr/*/rand_distr/index.html) crate provides other kinds of distributions. To use them, you instantiate a distribution, then sample from that distribution using [Distribution::sample](https://docs.rs/rand/*/rand/distributions/trait.Distribution.html#tymethod.sample) with help of a random-number generator [rand::Rng](https://docs.rs/rand/*/rand/trait.Rng.html).

Reply:
Sorry i think the name of the tests is not completely descriptive, i try to test that the random generator is uniform by applying the [central limit theorem](https://en.wikipedia.org/wiki/Central_limit_theorem). What has a Gaussian shape is the distribution of the histogram counts for a large quantity of samples taken.  IDNK if this is a correct/good way to test it.

Reply:
But doesn't the page say the requirement is that the samples are identically distributed (as in, all samples have the same distribution) rather than the elements in the sample having uniform distribution? I think CLT applies for many distributions other than uniform, even if that is included in the set.

Reply:
After checking it I think I was wrong, CLT applies to the distribution of the samples mean not to the distribution of the samples. Therefore, doing that, even if the generator will generate a nonuniform distribution, the test will pass so as you say that applies  for many distributions of the samples. A correct way the correct way would have been to apply chi-square test. Thanks for that! Anyway, this PR will be closed without merge